Makefile

```diff
@@ -176,6 +183,12 @@ else
 docker: docker-multi-stage
 endif
 
+.PHONY: docker-e2e
+docker-e2e: ## Builds 'thanos' docker for e2e tests
+docker-e2e: build-e2e 
```

Comment:
Do we need to do `build-e2e` if it's part of the Dockerfile build?

Reply:
Yes, now this needs to be removed